crates/worker/src/partition_processor_manager.rs

```diff
@@ -103,6 +104,7 @@ pub struct PartitionProcessorManager<T> {
 
     persisted_lsns_rx: Option<watch::Receiver<BTreeMap<PartitionId, Lsn>>>,
     invokers_status_reader: MultiplexedInvokerStatusReader,
+    control_processors: Option<ControlProcessors>,
```

Comment:
Maybe `pending_control_processors`? And adding a comment explaining what this field contains.